factories/esSearcherFactory.js

```diff
@@ -176,10 +189,78 @@
         }
       }, function error(msg) {
         activeQueries.count--;
-        thisSearcher.inError = true;
+        self.inError = true;
         return $q.reject(msg);
       });
-    }
+    } // end of search()
+
+    function explainOther (otherQuery, fieldSpec) {
```

Comment:
I would prefer putting the explainOther stuff in an external service that uses the searchers with the normalDocsSvc. Is that possible?